src/toxext.h

```diff
@@ -97,24 +110,31 @@ struct ToxExtPacketList *toxext_packet_list_create(struct ToxExt *toxext,
  * Appends an extension message onto an existing packet. This is the endpoint
  * extensions should use when trying to send their own data.
  */
+ // "existing packet" -> "existing packet list"?
+ // "This is the endpoint extensions" -> "This is the endpoint the extensions"
+ // not sure about "endpoint" in general vs API or function :shrug:
 int toxext_packet_append(struct ToxExtPacketList *packet /*in/out*/,
 			 struct ToxExtExtension *extension, void const *data,
 			 size_t size);
 
 /**
  * Sends and frees an existing packet
  */
+ // "packet" -> "packet list"? was called a "packet list" in toxext_packet_list_create, and that seems more accurate
 int toxext_send(struct ToxExtPacketList *packet);
 
 /**
  * Determines if the input data is meant for ToxExt
  */
+ // maybe some clarification on "input data", I'm guessing this means tox custom packet data
```

Comment:
I actually have to double check my qtox implementation since it's been a while. But I believe the intent is that for each custom packet someone can call is_toxext_packet before deciding whether or not to handle it themselves or hand it off to us.

Reply:
That's what I saw in the qTox PR, will update the comment to specify